src/Aspire.Hosting/Dashboard/DashboardLifecycleHook.cs

```diff
@@ -154,4 +189,111 @@ private void ConfigureAspireDashboardResource(IResource dashboardResource)
             }
         }));
     }
+
+    private async Task WatchDashboardLogsAsync()
+    {
+        async Task<string?> GetDashboardResourceIdentifier()
+        {
+            string? dashboardResourceId = null;
+
+            try
+            {
+                // The dashboard resource isn't immediately available so watch for it.
+                // Wait for dashboard to startup and be reported before giving up.
+                using var cts = new CancellationTokenSource(TimeSpan.FromSeconds(30));
+
+                await foreach (var notification in resourceNotificationService.WatchAsync(cts.Token))
+                {
+                    // We want to capture the replica of the dashboard resource. This will allow us to get the logs for the dashboard.
+                    // When the resource id is not the same as the resource name, it's the replica.
+                    if (notification.Resource.Name == KnownResourceNames.AspireDashboard && notification.ResourceId != notification.Resource.Name)
```

Comment:
```suggestion
                    if (string.Equals(notification.Resource.Name, KnownResourceNames.AspireDashboard, StringComparisons.ResourceName) && notification.ResourceId != notification.Resource.Name)
```
I'm not sure about resource ID. Is it case insensitive as well?

Reply:
We create the resource so no, it doesn't need to be insensitive.

Reply:
Not if there is an existing project with the name though. Other places check the name without case sensitivity: https://github.com/dotnet/aspire/blob/8e27fadd7db0a775bf23f38b9bc6a855671f101b/src/Aspire.Hosting/Dashboard/DashboardLifecycleHook.cs#L26

Reply:
True, I forgot we had this comparer. Fixed